packages/dds/merge-tree/src/test/attributionCollection.perf.spec.ts

```diff
@@ -138,7 +138,7 @@ function runAttributionCollectionSuite(
 }
 
 // Note: channel functionality is left unimplemented.
-export class TreeAttributionCollection implements IAttributionCollection<AttributionKey> {
+class TreeAttributionCollection implements IAttributionCollection<AttributionKey> {
```

Comment:
@Abe27342 this class came in one of your PRs, just checking that this is ok.

Reply:
yeah, this is fine. This code used to be what would be referenced in production if someone were to use attribution, but I re-implemented it using arrays for perf reasons (content doesn't get big enough that tree overhead is worth it) and kept around the old implementation to run the perf test suite against for comparison. I didn't bother switching the assert mechanism when I moved the class, but there's no real reason not to (at least after comparing initial perf numbers)